src/core/drive/morph_renderer.js

```diff
@@ -3,51 +3,32 @@ import { dispatch } from "../../util"
 import { PageRenderer } from "./page_renderer"
 
 export class MorphRenderer extends PageRenderer {
-  async render() {
-    if (this.willRender) await this.#morphBody()
-  }
-
-  get renderMethod() {
-    return "morph"
-  }
 
-  // Private
+  // Static methods — these helper methods are all static because morphing is done by turbo frames, which is outside of a PageView
```

Comment:
There is no change to the logic of any of these methods. They're simply converted to static methods so that they can be used by FrameController.